swagger.yaml

```diff
@@ -3649,6 +3662,17 @@ definitions:
         type: integer
       bootable:
         type: boolean
+      installation_eligibility:
```

Comment:
didn't we say that it should not be nil but just two fields `eligible` and `not_eligible_reasons` ?

Reply:
Didn't do the fix yet. Just pushed a rebase

Reply:
I've talked with @filanov about that. I personally prefer to have them under a struct - in order to group them together, because they're very related.

@filanov prefers they'd be directly under the disk struct (Just like you do).

Going my way would allow me to not change any existing code in this PR.
Going your way would require me to change a lot of code in this PR.

Seeing as I both prefer to have them grouped, and it would save me from doing more work - I'd need a good reason to not have them grouped. If you have convincing "technical" reasons for why it's bad to group them then I'll do the changes, but otherwise I'd prefer to keep it as it is.